### PR TITLE
Cloud Monitoring: add backend services out-of-the-box dashboard

### PR DESCRIPTION
**Pull request by naseemkullah**
**What this PR does / why we need it**:

This dashboard is based on one that I submitted to Cloud Moniroting Sample Dashboards, it is very useful to get per service golden signals.

https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/networking/https-lb-backend-services-monitoring.json

cc @papagian 

Special notes:
It would be great to share with the community and build on it, we are currently in the process of replacing our Cloud Monitoring dashbaords with grafana ones. The Cloud Monitoring plugin by Grafana is great btw

### CODE REVIEW

File: public/app/plugins/datasource/cloud-monitoring/dashboards/loadBalancerBackendServices.json [hunk 1]
```diff
@@ -0,0 +1,1538 @@
...
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${DS_GOOGLE_CLOUD MONITORING}",
```
Comment (papagian) [inline public/app/plugins/datasource/cloud-monitoring/dashboards/loadBalancerBackendServices.json:1497]:
```suggestion
        "datasource": "$datasource",
```

**Review by papagian — CHANGES_REQUESTED** [review]
Thank your contribution!
There is a remaining reference to `DS_GOOGLE_CLOUD MONITORING`.
Also you have to register you dashboard [in the plugin.json](https://github.com/grafana/grafana/blob/d0f52d5334d1e3623c7360d0291d29deb5d3f6eb/public/app/plugins/datasource/cloud-monitoring/plugin.json#L15) together with the others.

**Comment by naseemkullah** [comment]
Thanks @papagian! Corrections applied.

File: public/app/plugins/datasource/cloud-monitoring/dashboards/loadBalancerBackendServices.json [hunk 2]
```diff
@@ -0,0 +1,1538 @@
...
+    ]
+  },
+  "time": {
+    "from": "now-1h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Google Cloud HTTP(S) LB Backend Services",
+  "uid": "googlecloudhttpslbbackendservices",
```
Comment (naseemkullah) [inline public/app/plugins/datasource/cloud-monitoring/dashboards/loadBalancerBackendServices.json:1536]: BTW, would a generated hash preferred here?

**Comment by sunker** [comment]
Thank you so much for contributing @naseemkullah! We ended up modifying this dashboard just a tiny bit and then put it in [this](#29930) PR alongside a few other dashboards from Googles sample dashboard repo. Hope that's okay with you. Again, thank you so much for helping out on this!

**Comment by naseemkullah** [comment]
Sounds good, thanks.

**Pull request closed without merging**